app/services/app/app.ts

```diff
@@ -26,6 +26,7 @@ import { OutageNotificationsService } from 'services/outage-notifications';
 import { CrashReporterService } from 'services/crash-reporter';
 import { PlatformAppsService } from 'services/platform-apps';
 import { AnnouncementsService } from 'services/announcements';
+import uuid from 'uuid';
```

Comment:
We tend to use v4 uuids because they are purely random:
`import uuid from 'uuid/v4';`